dubbo-admin-server/src/main/java/org/apache/dubbo/admin/service/RegistryServerSync.java

```diff
@@ -126,19 +123,9 @@ public void notify(List<URL> urls) {
                 if (StringUtils.isEmpty(interfaceName)) {
                     interfaceName = getServiceInterface(url);
                 }
-                Map<String, Map<String, URL>> services = categories.get(category);
-                if (services == null) {
-                    services = new HashMap<>();
-                    categories.put(category, services);
-                }
-                String group = url.getUrlParam().getParameter(Constants.GROUP_KEY);
-                String version = url.getUrlParam().getParameter(Constants.VERSION_KEY);
-                String service = BaseServiceMetadata.buildServiceKey(getServiceInterface(url), group, version);
-                Map<String, URL> ids = services.get(service);
-                if (ids == null) {
-                    ids = new HashMap<>();
-                    services.put(service, ids);
-                }
+                Map<String, Map<String, URL>> services = categories.computeIfAbsent(category, k -> new HashMap<>());
+                String service = url.getServiceKey();
```

Comment:
Can't use `service key` because group and version use consumer URL parameter is "*".

Reply:
@haoyann get, please code review again and make CI rerun